Header header

#motor speeds (rad/s)
float64 motor1
float64 motor2
float64 motor3
float64 motor4

#moment arm and prop kf, kt
float64 d
float64 kf
float64 kt

